Sources/Hyperwallet.swift

```diff
@@ -907,6 +907,29 @@ public final class Hyperwallet: NSObject {
                                        completionHandler: transferMethodConfigurationFieldResponseHandler(completion))
     }
 
+    /// Returns the transfer method update configuration field set for the User that is associated with the
+    /// authentication
+    /// token returned from
+    /// `HyperwalletAuthenticationTokenProvider.retrieveAuthenticationToken(_ : @escaping CompletionHandler)`.
+    ///
+    /// The `completion: @escaping (HyperwalletTransferMethodUpdateConfigurationField?, HyperwalletErrorType?) -> Void`
+    /// that is passed in to this method invocation will receive the successful
+    /// response(HyperwalletTransferMethodUpdateConfigurationField) or error(HyperwalletErrorType) from processing the
+    /// request.
+    ///
+    /// This function will request a new authentication token via `HyperwalletAuthenticationTokenProvider`
+    /// if the current one is expired or is about to expire.
+    ///
+    /// - Parameters:
+    ///   - request: containing transfer method token to be updated
```

Comment:
Kindly elaborate this `request` parameter. You can take a look at `listTransferMethods` for reference

Reply:
Will refer and update it.

Reply:
request: GraphQL query  containing  transfer method token  and required configuration fields